ext/intl/listformatter/listformatter.stub.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/** @generate-class-entries */
+
+/** @not-serializable */
+/** @strict-properties */
```

Comment:
```suggestion
/**
 * @not-serializable
 * @strict-properties
 */
```
To make it more clear that these two belong together.